recipe/meta.yaml

```diff
@@ -11,29 +12,60 @@ source:
     - have_openssl.patch
 
 build:
-  number: 2
-
-requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - postgresql                             # [build_platform != target_platform]
-    - {{ compiler('c') }}
-  host:
-    - python
-    - pip
-    - openssl  # [not linux]
-    - postgresql
-  run:
-    - python
-    # libpq is a repackaging of only the library to connect to postgres
-    - libpq
-    - openssl  # [not linux]
-
-test:
-  imports:
-    - psycopg2
-    - psycopg2._psycopg
+  number: 3
+
+outputs:
+  - name: {{ name }}
+    script: install_psycopg2.sh  # [unix]
+    script: install_psycopg2.bat  # [win]
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - postgresql                             # [build_platform != target_platform]
+        - {{ compiler('c') }}
+      host:
+        - python
+        - pip
+        - openssl  # [not linux]
+        - postgresql
+      run:
+        - python
+        # libpq is a repackaging of only the library to connect to postgres
+        - libpq
+        - openssl  # [not linux]
+
+    test:
+      imports:
+        - psycopg2
+        - psycopg2._psycopg
+      commands:
+        - pip check
+      requires:
+        - pip
+
+  # alternative name for the package, patched to rename the package in the
+  # metadata so pip check will pass for packages that depend on pyscopg2-binary
+  - name: psycopg2-binary
+    script: install_psycopg2-binary.sh  # [unix]
```

Comment:
could this be `noarch`?

Reply:
That's a tricky one.  It could work if we add a skip to the build for all but one of he CI builds.  I'll see if that's possible.

Reply:
Okay, I've switched to noarch.  I've verified that only the linux py 38 build produces `psycopg2-binary` as an output and that it's not pegged to a specific python version.